### PR TITLE
fix universal contract scanning error

### PR DESCRIPTION
**Pull request by zorjak**

### CODE REVIEW

File: internal/platform/scan/scan.go [hunk 1]
```diff
@@ -191,12 +191,16 @@ func (s scanner) ScanNewUniversalEvents(ctx context.Context, fromBlock, toBlock
 		return nil, fmt.Errorf("error instantiating ABI: %w", err)
 	}
 
+	slog.Debug("event logs received", "logs", len(eventLogs))
```
Comment (kudlaj) [inline internal/platform/scan/scan.go]: I don't think this log is necessary

File: internal/platform/scan/scan.go [hunk 2]
```diff
@@ -191,12 +191,16 @@ func (s scanner) ScanNewUniversalEvents(ctx context.Context, fromBlock, toBlock
...
 	contracts := make([]EventNewERC721Universal, 0)
 	for i := range eventLogs {
 		if len(eventLogs[i].Topics) == 0 {
 			continue
 		}
-
+		if len(eventLogs[i].Data) == 0 {
```
Comment (kudlaj) [inline internal/platform/scan/scan.go:199]: maybe we should move this to line 205 and print an WARN if it is an universal contract without data

File: internal/platform/scan/scan.go (hunk 1 above)
Comment (luispdm) [reply internal/platform/scan/scan.go:194]: agree

Comment (luispdm) [reply internal/platform/scan/scan.go]: agree

File: internal/platform/scan/scan.go (hunk 2 above)
Comment (luispdm) [reply internal/platform/scan/scan.go:199]: I don't think that is necessary, I would rather unify the if conditions:
```go
if len(eventLogs[i].Topics) > 0 && len(eventLogs[i].Data) > 0 && eventLogs[i].Topics[0].Hex() == eventNewERC721UniversalSigHash {
    newERC721Universal, err := parseNewERC721Universal(&eventLogs[i], &contractAbi)
    if err != nil {
        return nil, err
    }
    slog.Info("received event", eventNewERC721Universal, newERC721Universal)
    contracts = append(contracts, newERC721Universal)
}
```

**Review by kudlaj — APPROVED** [review]

File: internal/platform/scan/scan.go (hunk 1 above)
Comment (zorjak) [reply internal/platform/scan/scan.go]: removed

File: internal/platform/scan/scan.go (hunk 2 above)
Comment (zorjak) [reply internal/platform/scan/scan.go:199]: I prefere separate checks in validations. if you don't insist I would keep it as it is.

**Review by luispdm — APPROVED** [review]

**Pull request merged**